scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -73,19 +73,70 @@ def remove_too_many_alt_allele_sites(vds):
 
 
 
+def annotate_adj(
+        mt: hl.MatrixTable,
+        adj_gq: int = 30,
+        adj_ab: float = 0.2,
+) -> hl.MatrixTable:
+    """
+    Annotate genotypes with adj criteria (assumes diploid).
+    Defaults correspond to gnomAD values.
```

Comment:
Not sure if it matters, i.e the docs are surfaced anywhere, or if this means default to meet gnomAD standards but this seems like it could cause a little confusion since gnomAD's defaults are different.